--- a/src/mesa/main/extensions_table.h
+++ b/src/mesa/main/extensions_table.h
@@ -341,7 +341,7 @@
 EXT(EXT_texture_lod_bias                    , dummy_true                             , GLL,  x , ES1,  x , 1999)
 EXT(EXT_texture_mirror_clamp                , EXT_texture_mirror_clamp               , GLL, GLC,  x ,  x , 2004)
 EXT(EXT_texture_mirror_clamp_to_edge        , ARB_texture_mirror_clamp_to_edge       ,  x ,  x ,  x , ES2, 2017)
-EXT(EXT_texture_norm16                      , EXT_texture_norm16                     ,  x ,  x ,  x ,  31, 2014)
+EXT(EXT_texture_norm16                      , EXT_texture_norm16                     ,  x ,  x ,  x ,  30, 2014)
 EXT(EXT_texture_object                      , dummy_true                             , GLL,  x ,  x ,  x , 1995)
 EXT(EXT_texture_query_lod                   , ARB_texture_query_lod                  ,  x ,  x ,  x ,  30, 2019)
 EXT(EXT_texture_rectangle                   , NV_texture_rectangle                   , GLL,  x ,  x ,  x , 2004)
